prow/scripts/resources/reconciler/shoot-template.yaml

```diff
@@ -25,7 +25,7 @@ spec:
     - machine:
         image:
           name: gardenlinux
-          version: 576.1.0
+          version: 576.9.0
```

Comment:
.9.0 is a preview version not fully supported yet

Reply:
which version is current support one?

Reply:
Se the gardener UI: 576.8.0